qlib/rl/from_neutrader/__init__.py

```diff
@@ -0,0 +1,2 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
I think `from_neutrader` should be renamed.

I remember I raised a similar comment once, but somehow I couldn't find it any more...